ocs_ci/ocs/resources/pvc.py

```diff
@@ -131,3 +131,30 @@ def create_multiple_pvc(number_of_pvc=1, pvc_data=None):
         pvc_objs.append(pvc_obj)
         log.info(f'Created Persistent Volume Claim {pvc_name}')
     return pvc_objs
+
+
+def get_pv_for_pvc(pvc_obj):
```

Comment:
I suggest to move this function under PVC class so we'll be able to get the backed pv of the pvc as a class property. Something like this:

```python
    @property
    def backed_pv_obj(self):
        self.reload()
        data = dict()
        data['api_version'] = self.api_version
        data['kind'] = 'PersistentVolume'
        data['metadata'] = {
            'name': self.backed_pv, 'namespace': self.namespace
        }
        pv_obj = OCS(**data)
        pv_obj.reload()
        return pv_obj
```

Reply:
ack

Reply:
Done